_aggregations/metric/stats.md

```diff
@@ -9,37 +9,166 @@ redirect_from:
 
 # Stats aggregations
 
-The `stats` metric is a multi-value metric aggregation that returns all basic metrics such as `min`, `max`, `sum`, `avg`, and `value_count` in one aggregation query.
+The `stats` aggregation is a multi-value metric aggregation that computes a summary of numeric data. This aggregation is useful for quick understanding of the distribution of a numeric fields. It can operate [directly on a field](#computing-stats-on-electricity-usage), apply a [script to derive the values](#using-a-script-to-compute-derived-values), or [handle documents with missing fields](#handling-documents-with-missing-fields). The `stats` aggregation returns five values:
 
-The following example returns the basic stats for the `taxful_total_price` field:
+* `count`: The number of values collected
+* `min`: The lowest value
+* `max`: The highest value
+* `sum`: The total of all values
+* `avg`: The average of the values (sum divided by count)
+
+
+## Example
+
+The following example computes `stats` aggregation on electricity usage.
+
+### Computing stats on electricity usage
+
+Create an index named `power_usage` and add documents where each document contains the number of kilowatt-hours (kWh) consumed during a given hour using the following request:
+
+```json
+PUT /power_usage/_bulk?refresh=true
+{"index": {}}
+{"device_id": "A1", "kwh": 1.2}
+{"index": {}}
+{"device_id": "A2", "kwh": 0.7}
+{"index": {}}
+{"device_id": "A3", "kwh": 1.5}
+```
+{% include copy-curl.html %}
+
+To compute statistics on the `kwh` field across all documents, use the following aggregation request which suppresses document hits by setting `size` to `0`, and defines a `stats` aggregation named `consumption_stats` over the `kwh` field:
+
+```json
+GET /power_usage/_search
```

Comment:
it would be nice to also add an example to nested term aggregation having stats for each bucket.